openfeature/provider/_registry.py

```diff
@@ -66,8 +66,6 @@ def shutdown(self) -> None:
 
     def _get_evaluation_context(self) -> EvaluationContext:
         # imported here to avoid circular imports
```

Comment:
I think this comment can go, too - it was only relevant to the import

Reply:
very true 😅